### ISSUE TITLE
Scraper jobs cannot start in environments where resource specification is mandatory

### ISSUE THREAD

**Issue by crdnb** [kubeshop/testkube#2942]
**Describe the bug**
Testkube jobs in general are missing memory and cpu resource specification. In clusters, where a resource definition is mandatory, these pods do not run:

`Error creating: pods "63a45ce3bcb3bf9d0d9a04ea-scraper-6j2qc" is forbidden: [minimum cpu usage per Pod is 1m. No request is specified, minimum memory usage per Pod is 1Mi. No request is specified, maximum memory usage per Pod is 12Gi. No limit is specified, maximum cpu usage per Pod is 2. No limit is specified]`

For executor jobs, job-templates can be used to patch the resource field of jobs / containers respectively. I did not find a way to patch resource specification to a scraper job.

Is it already possible to specify compute and memory resources for scraper jobs or jobs in general? If not, it would be great to have corresponding helm value fields to configure the resources as they already exists for many of the other components. Otherwise scraper jobs are not able to run.

**To Reproduce**
1. Run Testkube on a Kubernetes cluster, where resource specification (memory and cpu requests and limits) are mandatory
2. Watch the events, where you can see, that the pods without resource specification are not allowed to run

**Expected behavior**
Either helm values to configure resources or job-template functionality also for scraper jobs.

**Version / Cluster**
- Which testkube version? v1.8.7
- What Kubernetes cluster? OpenShift
- What Kubernetes version? v1.23

**Screenshots**
If applicable, add CLI commands/output to help explain your problem.
 
**Additional context**

**Comment by vsukhin** [comment]
thank you @crdnb When I was implementing this feature https://kubeshop.github.io/testkube/using-testkube/tests/tests-creating#changing-the-default-job-template-used-for-test-execution, I had a thought, that someone would also need it for scraper template and having it for executor jobs was not enough.
Still template is in helm chart https://github.com/kubeshop/helm-charts/blob/main/charts/testkube-api/job-scraper-template.yml

**Comment by crdnb** [comment]
Hi @vsukhin 
Thank you very much for your reply. I found the template for the scraper job but I did not manage to apply it. Can you give me a hint please? :)
Thank you very much

**Comment by vsukhin** [comment]
You can copy helm chart to your local folder and replace scraper template there. then run helm chart from local folder or fork repo,if you prefer git approach. still might be a proper solution to support scraper template extensions at test crd level

**Comment by crdnb** [comment]
Thank you very much for your answer. I did not figure out that the template is put into an env variable of the testkube-api deployment. So now I understand how it works.

I'll close this case, as the template extensions at test crd level is covered by the enhancement I already opened.

An answer that would help me a lot now would be, if it is possible, to define the bucket, where testkube places the test artifacts. My S3 user is only allowed to write to a specific bucket and is not allowed to create new ones. So if all the tests or at least all instances of the same test type could write to the same bucket would help a lot. If you know the answer, I'd much appreciate your reply.

Thank you very much!
